jupyter_fsspec/handlers.py

```diff
@@ -556,13 +627,15 @@ def setup_handlers(web_app):
     route_fs_file_transfer = url_path_join(
         base_url, "jupyter_fsspec", "files", "transfer"
     )
+    route_fs_sync = url_path_join(base_url, "jupyter_fsspec", "sync")
```

Comment:
I don't really see why we need a separate URL here is we don't generally for other operations.